### DISCUSSION TITLE
Unable to relay

### DISCUSSION THREAD

**Discussion by lokival** [postalserver/postal#1719] [Help with using Postal] [answered]
Good Day,

Attempting to set postal to use an SMTP relay -

smtp_relays:
  hostname: my.relay.com
  port: 25
  ssl_mode: Auto

However when testing the above using the web interface the delivery fails -

NoMethodError: undefined method `hostname' for ["hostname", "my.relay.com"]:Array

How do I set postal to send mail via a relay host?

**Comment by willpower232** [comment]
are you able to provide a log entry with stack trace to see where this error came from?

**Comment by lokival** [comment, marked as answer]
Update,

While attempting to do the stack trace, tried changing the relay config from -

`smtp_relays:
    hostname: my.relay.com
    port: 25
    ssl_mode: Auto`


to 

> smtp_relays:

> -

>     hostname: my.relay.com

>     port: 25

>     ssl_mode: Auto`

** The dot above is supposed to be a dash (minus sign) but I can't seem to figure out how to change it.

Also changed the timezone on the containers per the instructions on this link -

https://github.com/moby/moby/issues/8838#issuecomment-117284347

After make the above two changes, email was delivered.

**Reply by willpower232** [reply]
Ah yes good spot, the config must be properly yaml formatted with `-` and spaces

https://github.com/postalserver/postal/blob/ea10ae18f10ec243c9e30a540a3de4d853155838/config/postal.defaults.yml#L74-L78
